Sources/PackageCollections/Signing/PackageCollectionSignature.swift

```diff
@@ -0,0 +1,16 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import Crypto
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import Security
+#else
+import CCryptoBoringSSL
+#endif
```

Comment:
This is how swift-crypto will be used. We will need the `Security` framework as well. I added it so we can test everything together.

Reply:
Great, this stub is really helpful to see how it gets used.